src/Microsoft.Identity.Web/Constants/ErrorMessage.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Web
+{
+    internal static class ErrorMessage
+    {
+        public const string TenantIdClaimNotPresentInToken = "Neither `tid` nor `tenantId` claim is present in the token obtained from Microsoft identity platform.";
+        public const string TokenIsNotJWTToken = "Token is not JWT token.";
+        public const string ProvideEitherScopeKeySectionOrScopes = "Either provide the '{0}' or the '{1}' to the 'AuthorizeForScopes'. ";
+        public const string ScopeKeySectionIsProvidedButNotPresentInTheServicesCollection = "The {0} is provided but the IConfiguration instance is not present in the services collection";
+        public const string NoScopesProvided = "no scopes provided in scopes...";
+        public const string NeitherScopeOrRolesClaimFoundInToken = "Neither scope or roles claim was found in the bearer token.";
+        public const string IssuerMetadataURLIsRequired = "Azure AD Issuer metadata address URL is required";
+        public const string NoMetadataDocumentRetrieverProvided = "No metadata document retriever is provided";
+        public const string IssuerDoesNotMatchValidIssuers = "Issuer: '{0}', does not match any of the valid issuers provided for this application.";
+        public const string ClientInfoReturnedFromServerIsNull = "client info returned from the server is null";
+        public const string MissingClientCredentials = "missing_client_credentials";
+        public const string DuplicateClientCredentials = "duplicate_client_credentials";
```

Comment:
These looks a bit like error codes, meaning something depends on these values being recognized? Perhaps those should be in a separate file.